src/multiworld.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#ifndef CATA_SRC_MULTIWORLD_H
+#define CATA_SRC_MULTIWORLD_H
+//trans people are so cool :)
+
+#include <climits>
+#include <cstdint>
+#include <iosfwd>
+#include <new>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "game.h"
+#include "calendar.h"
+#include "catacharset.h"
+#include "color.h"
+#include "damage.h"
+#include "translations.h"
+#include "type_id.h"
+#include "point.h"
+class multiworld
+{
+        //allow access to set_world_prefix
+        friend void game::unserialize( std::istream &fin, const cata_path &path );
+    public:
+        multiworld();
+        ~multiworld();
+        std::string get_world_prefix();
+        //currently just the player
+        bool travel_to_world( const std::string prefix );
```

Comment:
```suggestion
        bool travel_to_world( const &std::string prefix );
```

Not sure about astyle. Maybe it should be `string_view` too, I haven't used them yet.

Ofcourse, use in .cpp too.

Reply:
Or something like that.

https://stackoverflow.com/a/51353813/5057078

Reply:
Sorry about the `const & std::string` order :D